modules/database/main.tf

```diff
@@ -0,0 +1,43 @@
+resource "random_string" "master_password" {
+  length  = 32
+  special = false
+}
+
+resource "random_pet" "postgres" {
+  length = 2
+  keepers = {
+    namespace = var.database_version
+  }
+}
+
+locals {
+  database_name        = "ctrlplane"
+  master_username      = "ctrlplane"
+  master_password      = random_string.master_password.result
+  master_instance_name = "${var.namespace}-${random_pet.postgres.id}"
+}
+
+resource "google_sql_database_instance" "postgres" {
+  name             = local.master_instance_name
+  database_version = var.database_version
+
+  settings {
+    tier = var.tier
```

Comment:
technically we aren't doing anything with the tier but felt wrong to hardcode directly here

Reply:
I think you are doing this fine within the module, but you should pass this all the way through to the top level variables file so it is configurable.